README.md

```diff
@@ -148,7 +148,15 @@ See [Tag Driven Releasing](.travis/).
 
 ### Publishing Artifacts
 
-To publish signed artifacts, use `publishSigned` or `publishLocalSigned`.
+To publish signed artifacts, use `publishSigned` or `publishLocalSigned`. Given the interactive prompting of a password you should also ensure that the `gpg` command has the ability to write to the console. This can be achieved using the following `export`:
```

Comment:
Hi Christopher, I hope things are ok.
As per the change, I think the bit should actually go to the section below "PIN entry (passphrase entry)" before or after the pin entry using gpg-agent.